Updates for 1.1.1 Bug Fix

timgdavies opened the pull request:
I've been working through the 1.1.1 milestone issues and typographical/terminological errors reported through the French translation process.

I've tried to keep things to individual commits as much as possible. 

Outstanding issues from 1.1.1 are:

* Writing a helper script (and tests?) to make sure the enum lists for closed codelists in the schema match the codelist.csv files: #475 
* Checking the status of milestoneDocuments #489 (I believe this is mainly an extensions registry issue and I think @Bjwebb has already done something for this)
* Updating the build process to make sure the correct links are included for cross-references etc. #501 (This may not be a blocker to the bug-fix release, as it is more about the build process)

I believe this branch should be ready to merge after a review from @duncandewhurst and/or @jpmckinney and/or @kindly - and then for @jpmckinney to run the aspell process over this.

Commit:
Update levels.md

Aligned list  with OCDS - 1.1 - Basic, Intermediate, Advanced spreadsheet 

- Award: Basic
- Planning: Intermediate, Advanced

Commit:
Correct typos

Commit:
Fixing documentation and codelist typos reported during French translation

Commit:
BugFix to correct milestoneType codelist

Commit:
Merge branch '1.1-dev' of https://github.com/open-contracting/standard into 1.1-dev

Commit:
Merging 471-milestone-type. Closes #471

Commit:
Documentation typo fixes based on review of the french translation

Commit:
Fixing typos in schema based on feedback from French translation

Commit:
Adding currency codelist. Closes #422

Commit:
Updating definition of contrctSchedule in documentType codelist. Closes #473

Commit:
Removing required release from record-package-schema.json - closes #479

Commit:
Clarifying the definition of budget.amount. Addresses #460. (changes to budgetBreakdown extension still required)

Commit:
Updating extension page display #496

Commit:
Replacing organizationIdentifierRegistrationAgency_iati.csv with org-id.guide

timgdavies commented:
(Note, travis tests failing because I forgot to run the update to make validation schema; can fix that at next stage...)

jpmckinney commented on standard/docs/en/schema/codelists.md:
```diff
@@ -46,17 +46,20 @@ Item quantities may be provided using an established codelist for units of measu
+For information on how to get new additions made to this list, see the [org-id.guide handbook](http://docs.org-id.guide/en/latest/contribute/)
 
-This list can be extended in consultation with IATI. 
+(**Update:** This list was formerly maintained by the International Aid Transparency Initiative and contained in OCDS docs as organizationIdentifierRegistrationAgency_iati.csv. This was removed in OCDS 1.1.1)
```
I think we use 'documentation' instead of 'docs'.

timgdavies replied:
Good spot

jpmckinney commented on standard/schema/codelists/awardCriteria.csv:
```diff
@@ -1,6 +1,6 @@
 Code,Title,Description,Deprecated
 priceOnly,Price only,"The award will be made to the qualified bid with the lowest price."
-costOnly,costOnly,"The award will be made to the qualified bid demonstrating the lowest overall cost. A cost assessment could cover the entire monetary implications of the proposal, including the price paid to the supplier and the running costs, switching costs or other non-price costs of choosing a particular option."
+costOnly,Cost only,"The award will be made to the qualified bid demonstrating the lowest overall cost. A cost assessment could cover the entire monetary implications of the proposal, including the price paid to the supplier and the running costs, switching costs or other non-price costs of choosing a particular option."
```
The other titles (except for 'Price only') have each word capitalized.

timgdavies replied:
Updating

jpmckinney commented on standard/schema/release-schema.json:
```diff
@@ -1042,7 +1032,7 @@
         "amount": {
           "title": "Amount",
-          "description": "The value of the budget line item. A negative value indicates anticipated income to the budget as a result of this contracting process, rather than expenditure.",
+          "description": "The value reserved in the budget for this contracting process. A negative value indicates anticipated income to the budget as a result of this contracting process, rather than expenditure. Where the budget is drawn from multiple sources, the budget breakdown extensions can be used.",
```
Shouldn't extension be singular here?

timgdavies replied:
Will update.

jpmckinney commented on standard/schema/release-schema.json:
```diff
@@ -1598,13 +1588,11 @@
         "scheme": {
           "title": "Scheme",
-          "description": "Organization identifiers be drawn from an existing identification scheme. This field is used to indicate the scheme or codelist in which the identifier will be found. This value should be drawn from the [Organization Identifier Scheme](http://standard.open-contracting.org/1.1/en/schema/codelists/#organization-identifier-scheme).",
+          "description": "Organization identifiers should be drawn from an existing organization identifier list. The scheme field is used to indicate the list or register from which the identifier is drawn. This value should be drawn from the [Organization Identifier Scheme](http://standard.open-contracting.org/1.1/en/schema/codelists/#organization-identifier-scheme) codelist.",
```
We use 'organization identifier list' here but elsewhere we use 'organization identifier scheme'. I wonder whether 'codelist' is redundant with 'scheme' in the last sentence.

timgdavies replied:
org-id.guide uses the language of lists, we have language of schema, and it's a codelist (sometimes talked about as a register). 

We're getting towards harmonized language... but I wanted to include each synonym in case.

jpmckinney replied:
Sounds good.

jpmckinney reviewed (commented):
I made a few copy-edit comments.

My only larger comment is: do the changes to the codelists invalidate any existing data? I think many of the codelists are open; for example, the changes to `milestoneType` won't invalidate data, though it should be highlighted in the upgrade notes so that implementers can switch from 'planning' and 'adjudication' to 'preProcurement' and 'assessment' if they choose. Assuming implementers were respecting the description of the `currency` field, there will be no invalidation there either, but I'm not sure whether that "if" is satisfied.

I believe there's a typo in the filename `locationGazeteers.csv`

Also, where are the upgrade notes - will they be updated in a separate PR?

Commit:
Merge branch '1.1-dev' into 1.1.1-dev

Commit:
Removing minLength, required from OrganizationReference.id Closes #484

Commit:
Fixing documentation build errors

Commit:
Making roles nullable. Closes #251

timgdavies committed:
Merge pull request #480 from open-contracting/bia-patch-1.1

Update levels.md

Commit:
Updating documentType contractSchedules codelist value. Closes #473.

Commit:
Fixing case in codelist files and schema opencodelist -> openCodelist

Commit:
Adding script to keep closed codelists in sync

Commit:
Updated validation schema and codelist script formatting

Commit:
Rename file for consistency

timgdavies committed:
Merge pull request #509 from open-contracting/update-enums

Adding script to keep closed codelists in sync

Commit:
Minor typographic updates

Commit:
Fix merge after #497

Commit:
ISO 639-1 has letters not digits

Commit:
Make references to ISO 639-1 consistent with other ISO references (i.e. no character between 'ISO' and number).

Commit:
Use American English instead of British English in text, unless quoting another source

Commit:
Another pass through spellings

Commit:
Fix case of some proper nouns

timgdavies committed:
Merge branch '1.1.1-dev' into typos

timgdavies committed:
Merge pull request #495 from jpmckinney/typos

Correct typos

Commit:
Merge branch '1.1.1-dev' of https://github.com/open-contracting/standard into 1.1.1-dev

Commit:
Change case of 'OCID' to 'ocid' and change 'meta-data' to 'metadata'

Commit:
Fix merge after pluralization of 'Extensions'

timgdavies committed:
Merge pull request #511 from jpmckinney/typos

'OCID' and 'metadata' changes / typos

Commit:
Merge branch '1.1.1-dev' of https://github.com/open-contracting/standard into 1.1.1-dev

Commit:
Adding changelog

Commit:
Rephrasing changelog

Commit:
Updating versioned release schema

duncandewhurst committed:
Update urls in changelog.md

All docs links the changelog were pointing to 1.1-dev

This update points links in the 1.1.1 section to 1.1.1-dev and links in the 1.1.0 section to 1.1

Commit:
Updating link to new basic, intermediate, advanced table

duncandewhurst committed:
Update changelog.md

Fix incorrect link to issue #475

duncandewhurst committed:
Add change to org id scheme codelist description to changelog

duncandewhurst committed:
Update links in ES changelog.po

Update /1.1-dev/ to /1.1/ in 1.1.0 changelog

duncandewhurst committed:
Update links in FR changelog.po

Update links from /1.1-dev/ to /1.1/ in 1.1.0 changelog

timgdavies committed:
Merge pull request #513 from open-contracting/fix-changelog-links

Updates to changelog.md

Commit:
Updating to 1__1__1 and minor documentation layout updates. Closes #510

Bjwebb committed:
Document tx push -t clobbering behaviour in README

https://github.com/open-contracting/standard/pull/495#issuecomment-317686611

timgdavies commented:
I'm closing this request, and will create a clean pull request that can be used for discussion during the 7-day review period for updates.
